airflow/api_internal/endpoints/rpc_api_endpoint.py

```diff
@@ -121,7 +121,7 @@ def initialize_method_map() -> dict[str, Callable]:
         MetastoreBackend._fetch_variable,
         XCom.get_value,
         XCom.get_one,
-        XCom.get_many,
+        # XCom.get_many, # Not supported because it returns query
```

Comment:
Should we remove it then? Or fix it? Or you want to keep it commented out to explain why it is not supported?

Reply:
It's "Maybe some time we will fix it" - I want to leave enough traces to show  where we are with AIP-44.